.github/workflows/api-tests.yml

```diff
@@ -29,7 +40,9 @@ jobs:
           DATACITE_PASSWORD: ${{ secrets.DATACITE_PASSWORD }}
           ORCID_ID: ${{ secrets.ORCID_ID }}
           ORCID_SECRET: ${{ secrets.ORCID_SECRET }}
-        run: docker compose -f "docker-compose.yml" up -d --build
+        run: |
+          export SERVERLESS_LICENSE_KEY=`aws ssm get-parameter --name ${{ secrets.SERVERLESS_LICENSE_KEY_SSM_PARAMETER_ARN }} --query "Parameter.Value" --output text`
+          docker compose -f "docker-compose.yml" up -d --build
```

Comment:
I know this line didn't change, but do you need to specify the yml file? Shouldn't it pick it up automatically?

Reply:
Good point, I will try it without